cases/util/responseLogCreator.js

```diff
@@ -0,0 +1,13 @@
+
+export const createLog = (response, responseBody) => {
+  
+  const rawHeaders = response.headers.raw();
+  const formattedHeaders = Object.keys(rawHeaders).map(key => `${key}: ${rawHeaders[key]}`).join("\n")
+  
+  var log = "\nRESPONSE ⬇️"
```

Comment:
You can use string interpolation instead of concatenation to make this a bit cleaner

```
var log = `RESPONSE
STATUS: ${response.status}
HEADERS: ${formattedHeaders}
BODY: ${JSON.stringify(responseBody, null, 2)}`;